src/vivarium_conic_lsff/components/observers.py

```diff
@@ -132,4 +132,98 @@ def get_transition_count(pop, config, disease, transition, event_time, age_bins)
                                                         year=event_time.year)
     base_filter = QueryString('')
     transition_count = get_group_counts(transitioned_pop, base_filter, base_key, config, age_bins)
-    return transition_count
\ No newline at end of file
+    return transition_count
+
+
+
+class LiveBirthWithNTDObserver:
+    """Observes births and births with neural tube defects. Output can be stratified
+    by year and by sex.
+    """
+    configuration_defaults = {
+        'metrics': {
+            'disease_observer': {
+                'by_year': True,
+                'by_sex': True,
+            }
+        }
+    }
+
+    def __init__(self):
+        self.disease = project_globals.NTD_MODEL_NAME
+        self.configuration_defaults = {
+            'metrics': {f'{self.disease}_observer': LiveBirthWithNTDObserver.configuration_defaults['metrics'][
+                'disease_observer']}
+        }
+
+    @property
+    def name(self):
+        return f'live_birth_with_ntd_observer'
+
+    def setup(self, builder):
+        self.config = builder.configuration['metrics'][f'{self.disease}_observer'].to_dict()
+        self.live_birth_counts = Counter()
+        self.with_disease = Counter()
+        tmp = builder.configuration['time']['start']
+        self._reference_time = pd.Timestamp(tmp['year'], tmp['month'], tmp['day'])
+
+        columns_required = ['alive', f'{self.disease}', 'entrance_time']
+        if self.config['by_sex']:
+            columns_required.append('sex')
+        self.population_view = builder.population.get_view(columns_required)
+
+        builder.value.register_value_modifier('metrics', self.metrics)
+        builder.event.register_listener('collect_metrics', self.on_collect_metrics)
+
+    def on_collect_metrics(self, event):
+        pop = self.population_view.get(event.index)
+        self.live_birth_counts.update(
+            get_live_births(pop, self.config, self._reference_time, event.time.year))
+        self.with_disease.update(
+            get_with_ntd_births(pop, self.config, self._reference_time, event.time.year))
+        self._reference_time = pd.Timestamp(event.time.year, event.time.month, event.time.day)
+
+    def metrics(self, index, metrics):
+        metrics.update(self.live_birth_counts)
+        metrics.update(self.with_disease)
+        return metrics
+
+    def __repr__(self):
+        return f"DiseaseObserver({self.disease})"
+
+
+def get_live_births(pop, config, reference_time, year):
+    retval = {}
+    filter = QueryString(f'alive == "alive" and entrance_time >= "{reference_time}"')
+    base_key = get_output_template(False, **config).substitute(
+        measure=f'live_births', year=year if config['by_year'] else None)
```

Comment:
Always pass the year.  The output template is smart and will ignore it if the config is not by year.